avaframe/ana5Utils/distanceTimeAnalysisCfg.ini

```diff
@@ -0,0 +1,48 @@
+### Config File - This file contains the main settings for the range time diagram in distanceTimeAnalysis
+## Copy to  local_distanceTimeAnalysisCfg.ini and set you parameters
+# This file is part of Avaframe.
+
+
+[GENERAL]
+# result type used for creating range time diagram e.g. FD, FV, ....
+rangeTimeResType = FD
+# time steps used for creating diagram
+tSteps = 0:1
+# threshold - below this result data is ignored - used to create mask
+# average value created of where there is avalanche flow (threshold > 0.0) or including all cells in
+# range gate (also the ones where there is no snow flowing - threshold = 0.0)
```

Comment:
should we do it the other way around so that we keep field>threshold. Then we do not have to do this trick of putting 0.000001

Reply:
any suggestions?

Reply:
instead of masking cell with value <= threshold keep cells with value > threshold

Reply:
see my comment below

Reply:
now we make a mask flowF > threshold and invert it